clients/drcachesim/tools/filter/record_filter.h

```diff
@@ -70,6 +70,16 @@ class record_filter_t : public record_analysis_tool_t {
          * #trace_entry_t, hence the vector.
          */
         std::vector<trace_entry_t> *last_encoding;
+
+        /**
+         * Gives filters access to dcontext_t.
+         */
+        /* xref i#6690 i#1595: dcontext_t is not entirely thread-safe. AArch32 encoding
```

Comment:
Make this part of the doc above. That is: drop the `/` on this line and above.

Reply:
Done.